### ISSUE TITLE
Media Gallery

### ISSUE THREAD

**Issue by ssanchezm** [ssanchezm/testPM#1]
### Background, context and strategic fit
Brief introduction. See first sections of XO specs.

### Must-haves
_describe the list of items that absolutely must be finished, keep the list short_
- [x] item 1
- [x] item 2
**I don't like this list. I really think that features (or subfeatures) must be properly defined. IMHO all that is specified is a must have**

### Nice-to-haves
_describe the list of items would be nice to be done, you may keep the list long_
- [ ] item 1
- [ ] item 2
**I would remove this section. _Nice to have_ things must be kept in backlog, not in feature specifications** 

### Not doing and trade-offs
_describe things that are out-of-scope for this project and trade-offs of the solution_
Ok.

### Specification, mockups, workflows and flowcharts
_if applicable, provide links to_
* specification link
* mockups link
Ok.

## Test plan
_describe the set of tests to pass for the feature to be released_
- [ ] test 1
- [ ] test 2
**Check if the test plan should be created per issue or per task.**

## Timeline
* Specify a date when the feature needs to be ready (if applicable)
* If spilled over multiple release cycles, specify deadlines for each cycle
* Specify intermediate steps - beta versions, deployment dates
_Dates to be added to the Engineering calendar._
**This is a good starting point. We might try to introduce an SE7 or SE15 to STAGE.**

## Status
- [ ] Slack channel created (_if applicable_)
- [ ] Specification completed
- [ ] Implementation
- [ ] Documentation
- [ ] Testing
  - [ ] Code review
  - [ ] Testlio (_if applicable_)
  - [ ] Staging servers (_if applicable_)
  - [ ] Pilot event (_if applicable_)

## Dependencies
- [ ] links to a dependent issues/projects that need to be completed before this one
**This section can be improved. Check XO specs.**

## Links
- Links to issues in respective repos (iOS/Android/etc.)
- Links to wiki pages
- Links to relevant documents or assets
**This section can be improved. Check XO specs.**
